module/VuFind/config/module.config.php

```diff
@@ -333,6 +333,7 @@
     'service_manager' => [
         'allow_override' => true,
         'factories' => [
+            'MarkdownConverter' => 'VuFind\Service\MarkdownFactory',
```

Comment:
Service names should correspond to class or interface names, so I think this should actually be `League\CommonMark\GithubFlavoredMarkdownConverter` (or maybe better yet, `\League\CommonMark\ConverterInterface`). If you think it's important to have a short service name, you can make `MarkdownConverter` an alias, though I think that's probably just extra unnecessary complexity.

Reply:
I'll go with `League\CommonMark\ConverterInterface`, thanks. I also don't think we need a short name here.